bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/ReadLacProcessorV3.java

```diff
@@ -74,10 +74,10 @@ private ReadLacResponse getReadLacResponse() {
             }
         } catch (Bookie.NoLedgerException e) {
             status = StatusCode.ENOLEDGER;
-            logger.error("No ledger found while performing readLac from ledger: {}", ledgerId, e);
+            logger.warn("No ledger found while performing readLac from ledger: {}", ledgerId, e);
         } catch (IOException e) {
             status = StatusCode.EIO;
-            logger.error("IOException while performing readLac from ledger: {}", ledgerId);
+            logger.warn("IOException while performing readLac from ledger: {}", ledgerId);
```

Comment:
an IO error is a signal that something is going wrong.
I would keep it at "error" level

Reply:
yes, it should be error here,  i change it back to error.

There is another IOException error log, i think it maybe "warn" level, help to review. thanks

Reply:
the message is like this
`1)      03:32:11.130 [BookieReadThreadPool-OrderedExecutor-0-0] ERROR org.apache.bookkeeper.proto.ReadLacProcessorV3 - IOException while trying to read last entry: 2
org.apache.bookkeeper.bookie.Bookie$NoEntryException: Entry -1 not found in 2
`